CHANGES.rst

```diff
@@ -231,6 +231,8 @@ Bug fixes
 
 - ``astropy.io.fits``
 
+  - ``fitsrec.__del__`` does not print an array if it fails to delete ``_coldefs``. The AssertionError is replaced with a if-statement that checks if ``_coldefs`` exists before trying to delete it. [#4690]
```

Comment:
Please wrap this over multiple lines.  Although actually its text is too technical.  Instead you can just write something like
```
Fixed ``AssertionError`` that could occur after closing or deleted compressed image data.
```
Nothing technical about *how* it was fixed.